3515/1/files/gems_main.rb

```diff
@@ -0,0 +1,37 @@
+class GemsMain
```

Comment:
Try to name classes in way than reader can get logic what that class does. From `GemsMain` I don't understand what the job of that class